File: Platform/codesignal/Intro/29.chessBoardCellColor/solution.go
package main

import (
	"fmt"
	"math"
)

func solution(c1, c2 string) bool {
	return int(math.Abs(float64(c2[0]-c1[0]))) % 2 == int(math.Abs(float64(c2[1]-c1[1]))) % 2
}

func main() {
	c1 := "A1"
	c2 := "C3"

	fmt.Println(solution(c1, c2))
}

File: Platform/codesignal/Intro/29.chessBoardCellColor/solution.py
def solution(c1, c2):
  return (ord(c2[0]) - ord(c1[0])) % 2 == (int(c2[1]) - int(c1[1])) % 2



# print(solution("A1", "C3"))
print(solution("A1", "B2"))